tortoise_interpreter.py

```diff
@@ -0,0 +1,21 @@
+def main_tortoise(program):
+    COMMANDS = {'U': pen_up, 'D': pen_down, 'P 3': pen_three}
```

Comment:
This isn't sufficient for other tortoise programs. Consider this tortoise program:
```
P 2
U
W 3
P 5
S 4
D
```
What _should_ be printed when it's run through the interpreter? What _will_ happen if you run it now? What happens if you run `example-program.tortoise`?